### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LeonoorsCryptoman**
Closes: #XXX

## Context and purpose of the change
Added the instructions to set up a Stride node, since it was missing in the README.MD file on Github

## Brief Changelog
Added the instructions to the README.MD

## Author's Checklist
I have...
- [n/a] Run and PASSED locally all GAIA integration tests
- [n/a] If the change is contentful, I either:
    - [n/a] Added a new unit test OR 
    - [n/a] Added test cases to existing unit tests
- [n/a] OR this change is a trivial rework / code cleanup without any test coverage

If skipped any of the tests above, explain.
No code added, only change to the README.MD

## Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] manually tested (if applicable)
- [ ] confirmed the author wrote unit tests for new logic
- [ ] reviewed documentation exists and is accurate

## Documentation and Release Note

  - [n/a] Does this pull request introduce a new feature or user-facing behavior changes?  > No
  - [n/a] Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? > No
  - [n/a] This pull request updates existing proto field values (and require a backend and frontend migration)? > No
  - [n/a] Does this pull request change existing proto field names (and require a frontend migration)? > No
  How is the feature or change documented? 
      - [X] not applicable
      - [ ] jira ticket `XXX` 
      - [ ] specification (`x/<module>/spec/`) 
      - [ ] README.md 
      - [ ] not documented

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,126 @@ Stride is a blockchain ("zone") that provides liquidity for staked assets. Using
...
+sudo apt-get upgrade -y
+sudo apt-get install -y build-essential curl wget jq
+sudo su -c "echo 'fs.file-max = 65536' >> /etc/sysctl.conf"
+sudo sysctl -p
+```
+
+### Install go
+First remove any existing old Go installation:
+```
+sudo rm -rf /usr/local/go
```
Comment (asalzmann) [inline README.md:32]: This command feels slightly dangerous to run? Not sure

**Review by asalzmann — COMMENTED** [review]
Thanks for putting this together! 

I think these commands are linux specific, should we put this in a separate README that outlines running a node on linux?

File: README.md (hunk 1 above)
Comment (LeonoorsCryptoman) [reply README.md:32]: It only deletes an old folder of go (if it exists). Not dangerous, have done it a couple of times already :P
And the command following immediately is installing a newer version of go.

**Comment by LeonoorsCryptoman** [comment]
> Thanks for putting this together!
> 
> I think these commands are linux specific, should we put this in a separate README that outlines running a node on linux?
They are indeed Linux specific. Not sure if there are people running nodes on other OS though.
So can't judge if it is of added value to make a separate readme per OS

**Comment by asalzmann** [comment]
@vish-stride do you have thoughts?

File: README.md (hunk 1 above)
Comment (sampocs) [reply README.md:32]: Agreed on it not being dangerous - it's stated in the go installation [docs](https://go.dev/doc/install)

**Review by sampocs — APPROVED** [review]
This looks good to me! I agree with @asalzmann though in that you should specify that this is for linux only. You can even just add it to the section title: "## Setting up a node (on Linux)"

**Comment by vish-stride** [comment]
First off, thanks a lot for making this PR Leonoor's, I really appreciate it! Very much needed. 

Agreed with Aidan and Sam, we should add just a small disclaimer saying this is Linux only. I know it's very straightforward from looking at the code, but I think it's better to make it explicit.

Only thing from my end is I think this code would be better scoped to [this repo](https://github.com/Stride-Labs/testnet). It's fairly sparse right now, but I think it's reasonable to separate out "connecting to Stride chain" vs "Stride protocol code". For example, we could throw in helper scripts for RPC nodes, instructions to spin up a seed node, etc. 

Admittedly, the name of that repo is also outdated now. What do you all think about renaming it to something like "mainnet" or "node-setup"? Very open to other suggestions as well! 

@LeonoorsCryptoman , @asalzmann , and @sampocs, what do you think about moving these instructions to the other repo?

**Comment by sampocs** [comment]
@vish-stride In general, I like the idea of having a separate repo for node-related stuff! But considering everything in that repo is deprecated atm, and we don't have plans to add more resources in the short term, it might make more sense to include these instructions in this main repo for now. I also think, if I was a coming to Stride for the first time, I'd probably assume instructions were in this repo.

**Comment by LeonoorsCryptoman** [comment]
I think it might be good to place it under the testnet repo (and rename it to mainnet if it does not contain testnet files anyway).

In the main README.MD you could easily integrate a link to the right repo with just a brief explanation like; "if you want to setup your node for the Stride mainnet, find the relevant files and instructions here". Then you are good to go.

Main advantage is that also your main README.MD becomes a bit shorter, which is anyway a good thing imo.

**Comment by LeonoorsCryptoman** [comment]
1 check fails, but no clue how to fix. Any help?

**Comment by riley-stride** [comment]
Thanks for writing this @LeonoorsCryptoman. Great contribution. I copied it over to the `mainnet` repo (previously called `testnet`) [here](https://github.com/Stride-Labs/mainnet/blob/main/mainnet/README.md) and linked to it from that repo's main `README` [here](https://github.com/Stride-Labs/mainnet/blob/main/README.md). 

@LeonoorsCryptoman Would you mind updating this PR to simply add something like "if you want to setup your node for the Stride mainnet, find the relevant files and instructions here" to this repo's README, with a link to the mainnet repo? Then I'll approve this and we can merge it in. Thanks again for contributing!

**Comment by LeonoorsCryptoman** [comment]
Should be ready to go!

**Comment by LeonoorsCryptoman** [comment]
Still actions to take here?

**Comment by sampocs** [comment]
@LeonoorsCryptoman Yeah, I think based on @riley-stride's last point, we should remove the instructions in this readme and instead include a link to https://github.com/Stride-Labs/mainnet/blob/main/mainnet/README.md

> Would you mind updating this PR to simply add something like "if you want to setup your node for the Stride mainnet, find the relevant files and instructions here" to this repo's README, with a link to the mainnet repo? Then I'll approve this and we can merge it in. Thanks again for contributing!

File: README.md [hunk 2]
```diff
@@ -16,6 +16,129 @@ Stride is a blockchain ("zone") that provides liquidity for staked assets. Using
...
+sudo systemctl enable strided.service && sudo systemctl start strided.service
+```
+
+Monitor using:
+```
+systemctl status strided
+
+sudo journalctl -u strided -f
+```
+
```
Comment (sampocs) [inline README.md:141]: Replace with the following:

```
## Running a Mainnet Node
If you want to setup your node for the Stride mainnet, find the relevant files and instructions [here](https://github.com/Stride-Labs/mainnet/blob/main/mainnet/README.md)
```

**Comment by LeonoorsCryptoman** [comment]
Adjusted :)

**Pull request merged**